.github/workflows/msbuild.yml

```diff
@@ -53,13 +53,12 @@ jobs:
       run: dotnet pack --no-build dafny/Source/Dafny.sln
 
   check-deep-tests:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'run-deep-tests') }}
```

Comment:
```suggestion
    if: !contains(github.event.pull_request.labels.*.name, 'run-deep-tests')
```

Reply:
Nope, that's what I tried first, but then the value looks like a YAML tagged value: https://yaml.org/spec/1.2.2/#691-node-tags :)

Reply:
It worked before and the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif) also claim so.

Reply:
My bad, I just tried it. 😞